test/lint/lint-spelling.ignore-words.txt

```diff
@@ -11,5 +11,4 @@ nin
 ser
 unparseable
 unser
-useable
```

Comment:
https://www.merriam-webster.com/dictionary/useable

Also, if you want to modify libraries, you will have to do it upstream

Reply:
Most of the spelling in the codebase uses `usable` not `useable`.

<img width="1011" alt="Screen Shot 2021-05-07 at 9 51 10 pm" src="https://user-images.githubusercontent.com/418747/117445821-86f6e680-af7e-11eb-8497-6810336abf92.png">